src/taskpane/pages/dashboard.tsx

```diff
@@ -59,6 +60,55 @@ function Dashboard({ citeSupport }: DashboardProps): ReactElement {
       return { id: item.id };
     });
 
+  const unCheckAllCheckboxes = () => {
+    setItems((currentItems) => {
+      return currentItems.map(unCheckCheckbox);
+    });
+  };
+
+  const checkCitationItems = (itemId: Array<string>) => {
+    setItems((currentItems) => {
+      return currentItems.map((item) => {
+        if (item.id === itemId[0]) {
+          return { ...item, isSelected: true };
+        }
+        return item;
+      });
+    });
+  };
+
+  const getSelectedCitation = async (): Promise<number | void> => {
```

Comment:
The "get currently selected citation"-part (i.e. everything except the call to `checkCitationItems`) should move to the word-api and citation-support files.